engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/CreditsScreen.java

```diff
@@ -50,15 +51,28 @@ public void initialise() {
         creditsScroll = find("creditsScroll", UIScrollingText.class);
         if (creditsScroll != null) {
             ClassLoader classloader = getClass().getClassLoader();
-            InputStream is = classloader.getResourceAsStream("Credits.txt");
+            InputStream is = classloader.getResourceAsStream("Credits.md");
```

Comment:
Could you add a check for this resulting in a valid file? And if not then just show a simple warning message on the credits screen. Maybe something about it possibly being due to running from source where credits are unavailable

On a related note could you please throw an entry into the `.gitignore` to cover where this file would end up?